torch_geometric/nn/models/schnet.py

```diff
@@ -104,10 +102,13 @@ def __init__(self, hidden_channels: int = 128, num_filters: int = 128,
         self.std = std
         self.scale = None
 
-        atomic_mass = torch.from_numpy(ase.data.atomic_masses)
-        self.register_buffer('atomic_mass', atomic_mass)
+        if self.dipole:
+            import ase
 
-        self.embedding = Embedding(100, hidden_channels)
+            atomic_mass = torch.from_numpy(ase.data.atomic_masses)
+            self.register_buffer('atomic_mass', atomic_mass)
+
+        self.embedding = Embedding(100, hidden_channels, padding_idx=0)
```

Comment:
Can you clarify why we need `padding_idx=None`?

Reply:
We use padding atoms with atomic number == 0 to fill partial batches up to a fixed size.  The `padding_idx` option on the embedding could use better documentation but essentially:

* zeros out the embedding vector 
* detaches the corresponding row in the weight matrix from receiving weight updates.

Adding a comment in case that helps.

Reply:
Ok, but this might break for anyone how uses `0` as a regular atom type, right?

Reply:
I'm not aware of anyone using `SchNet` for non-molecular datasets so I thought this would be safe since atomic number zero doesn't map to any elements.